libnetwork/config/config.go

```diff
@@ -41,48 +29,31 @@ type DaemonCfg struct {
 	ClusterProvider        cluster.Provider
 	NetworkControlPlaneMTU int
 	DefaultAddressPool     []*ipamutils.NetworkToSplit
-}
-
-// LoadDefaultScopes loads default scope configs for scopes which
-// doesn't have explicit user specified configs.
-func (c *Config) LoadDefaultScopes(dataDir string) {
-	for k, v := range datastore.DefaultScopes(dataDir) {
-		if _, ok := c.Scopes[k]; !ok {
-			c.Scopes[k] = v
-		}
-	}
-}
-
-// ParseConfig parses the libnetwork configuration file
-func ParseConfig(tomlCfgFile string) (*Config, error) {
-	cfg := &Config{
-		Scopes: map[string]*datastore.ScopeCfg{},
-	}
-	data, err := os.ReadFile(tomlCfgFile)
-	if err != nil {
-		return nil, err
-	}
-	if err := toml.Unmarshal(data, cfg); err != nil {
-		return nil, err
-	}
-
-	cfg.LoadDefaultScopes(cfg.Daemon.DataDir)
-	return cfg, nil
+	Scopes                 map[string]*datastore.ScopeCfg
+	ActiveSandboxes        map[string]interface{}
+	PluginGetter           plugingetter.PluginGetter
 }
 
 // ParseConfigOptions parses the configuration options and returns
 // a reference to the corresponding Config structure
-func ParseConfigOptions(cfgOptions ...Option) *Config {
+func ParseConfigOptions(opts ...Option) *Config {
```

Comment:
Actually considering if we should rename this to `New()` - effectively it's a constructor for Config; WDYT?

Reply:
I think that's a great idea! 👍

Reply:
Done! 🎉